diff -uN distr/zchaff_utils.cpp win/zchaff_utils.cpp
--- distr/zchaff_utils.cpp	2006-09-27 17:43:51.000000000 +0200
+++ win/zchaff_utils.cpp	2006-09-27 17:43:33.000000000 +0200
@@ -37,8 +37,8 @@
 #include <stdarg.h>
 #include <stdlib.h>
 #include <unistd.h>
-#include <sys/time.h>
-#include <sys/resource.h>
+#include <time.h>
+
 
 int _global_debug_leveli = 0;
 
@@ -79,11 +79,5 @@
 }
 
 double get_cpu_time(void) {
-  double res;
-  struct rusage usage;
-  getrusage(RUSAGE_SELF, &usage);
-  res = usage.ru_utime.tv_usec + usage.ru_stime.tv_usec;
-  res *= 1e-6;
-  res += usage.ru_utime.tv_sec + usage.ru_stime.tv_sec;
-  return res;
+  return  (double)clock()/(double)(CLOCKS_PER_SEC);
 }
diff -uN distr/zverify_df.cpp win/zverify_df.cpp
--- distr/zverify_df.cpp	2006-09-27 17:43:51.000000000 +0200
+++ win/zverify_df.cpp	2006-09-27 17:43:33.000000000 +0200
@@ -33,8 +33,6 @@
 // of the possibility of those damages.
 // *********************************************************************
 
-#include <sys/time.h>
-#include <sys/resource.h>
 #include <vector>
 #include <set>
 #include <iostream>
